contracts/schemes/Auction4Reputation.sol

```diff
@@ -80,6 +83,7 @@ contract Auction4Reputation {
         auctionReputationReward = _auctionReputationReward;
         reputationRewardLeft = _auctionReputationReward.mul(_numberOfAuctions);
         redeemEnableTime = _redeemEnableTime;
+        agreementHash = _agreementHash;
```

Comment:
Would it be helpful, for each contract, to require that the hash be != 0x0?

A 0x0 hash would not fulfill the purpose of the feature.

Reply:
I think it is good yo have 0, for the people who dont want this feature.

Reply:
OK, I have previous heard the opposite argument for not undertaking other suggestions I've made about the bootstrapping contracts:  That no one else is ever going to use these contracts.  lol